astropy/io/fits/column.py

```diff
@@ -73,7 +73,7 @@
 
 # Maps FITS ASCII column format codes to the appropriate Python string
 # formatting codes for that type.
-ASCII2STR = {'A': 's', 'I': 'd', 'J': 'd', 'F': 'f', 'E': 'E', 'D': 'E'}
+ASCII2STR = {'A': '', 'I': 'd', 'J': 'd', 'F': 'f', 'E': 'E', 'D': 'E'}
```

Comment:
It does, as these characters are replaced in various formatting strings, and with the format() method it led to failures.